libs/@guardian/source-react-components-development-kitchen/src/file-input/FileInput.tsx

```diff
@@ -12,6 +12,7 @@ import {
 } from './styles';
 import type { Theme } from './theme';
 import type { FileInputProps } from './types';
+import { SupportingText } from 'libs/@guardian/source-react-components/src/label/SupportingText';
```

Comment:
I think this import is incorrect:

```suggestion
import { SupportingText } from '@guardian/source-react-components/src/label/SupportingText';
```

Reply:
Yeah i thought the same but interestingly removing `libs/` results in this error https://github.com/guardian/csnx/actions/runs/7904047399/job/21573404262?pr=1199#step:5:370